ARCharts/ARBarChart.swift

```diff
@@ -155,6 +161,11 @@ public class ARBarChart: SCNNode {
                     self.addLabel(forIndex: index, atXPosition: xPosition + xShift - 0.5 * barsWidth)
                 }
                 previousXPosition = xPosition
+                
+                if let animationType = animationType {
+                    let animationManager = AnimationManager(animationType: animationType, animationDuration: animationDuration)
+                    animationManager.addAnimation(toBarNode: barNode, atIndex: index, withBarHeight: barHeight, yPosition, opacity)
+                }
```

Comment:
Can we construct the `AnimationManager` only once, when the `animationType` gets set? I.e., make `animationManager` a private member on `ARBarChart`, which gets set on `didSet` on the `animationType`?